backend/api/v1/endpoints/documents.py

```
@@ -14,21 +16,32 @@
     current_user: dict = Depends(get_current_user),
-) -> dict:
+) -> JSONResponse:
     user_id = current_user.get("sub")
     documents = get_user_documents_from_vector_store(user_id)
-    return responses.JSONResponse(content={"documents": documents}, status_code=200)
+    return JSONResponse(content={"documents": documents}, status_code=200)
+
+
+@documents_router.delete("/{document_id}")
+async def delete_user_document(
+    current_user: dict = Depends(get_current_user),
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)